### DISCUSSION TITLE
Sustainability of running Uyuni Server on a RHEL base

### DISCUSSION THREAD

**Discussion by TrickPhysics** [sbluhm/uyuni#131] [General]
Hey there!  So I want to use RHEL 9 as the base OS for Uyuni, AND all the machines Uyuni will be managing are also RHEL 9.   The clients don't worry me, but I do have concerns that the server installation seems to heavily favor OpenSUSE (which is not an option for me).

Using your stuff here, I can get Uyuni running on RHEL 9, but am I constantly have to be tinkering on the back end to finesse each Uyuni update into my RHEL9 system?

Basically, what I'm asking is -- all the Uyuni components -- once I get and set up the Red Hat equivalent pieces should I be good to go?  Or is there SUSE-specific stuff like code or applications or packages that I'm always going to be up against?

Thanks!

**Comment by sbluhm** [comment]
Hello @TrickPhysics ,

I am/was running exactly the same configuration.

My instructions and setup should not have required any tinkering. The master code has continuously been adapted to support Enterprise Linux. I added all KNOWN required tinkering in the patch script until the fixes have been added upstream.
No SUSE related things had to be installed.

BUT keep in mind that:

1. I am not maintaining this project any more (for the reasons below). Last fully working release was Version 2024.02 (March 2024). Later versions/master did work but I never released the packages due to some package build error (unrelated to Uyuni) that required a manual workaround for each build. My master code is on a later state (probably June or July).
2. Uyuni standalone is deprecated and one of the next releases will be container only. So you have no choice but to move to containers.
3. Uyuni is already available as container and I am running it on Enterprise Linux 9 since its first release. So sticking to an officially maintained version is a lot easier and more stable.

Therefore I can only recommend to migrate to the fully Uyuni Project supported container version.

**Reply by TrickPhysics** [reply]
Oooooooohhhhh okay!  Well that actually works out great, I was wondering about the future direction, and was reading some conflicting stuff (always outdated stuff on the Internet, of course).

Thank you for explaining.  This is fantastic news.   I will go down the path of simply using a containerized Uyuni on our RHEL 9 host systems.  You're right, this DOES make everything much more sustainable and easier in the long run

Very much appreciate you taking the time to explain all of this, and for all of the work you did previously -- seeing the work you did to "middleware" between SUSE and RHEL was invaluable to my RHEL-centric experience and helped a ton.

Thank you so much!
